repos/rust-lang/flate2-rs.toml

```diff
@@ -0,0 +1,14 @@
+org = 'rust-lang'
+name = 'flate2-rs'
+description = 'DEFLATE, gzip, and zlib bindings for Rust'
+bots = []
+
+[access.teams]
+libs-contributors = 'maintain'
```

Comment:
Should this & others be libs-contributors or crate-maintainers?

(There's admittedly a lot of overlap between the two, and I'm on both so I don't particularly care, but the others were crate-maintainers)

Reply:
Whoops! 😅 Fixed.